Drivers/Src/BatteryBalancing.c

```diff
@@ -37,7 +40,7 @@ void Balancing_Balance(cell_asic Minions[]){
 	uint16_t lowest = GetMinimumVoltage(); //get lowest voltage 
```

Comment:
It won't let me leave a comment for this function since it wasn't changed, but in the for loop, shouldn't it start at 0 since the values are 0-indexed?

Reply:
The for loop starts at 1 instead of zero because the line above the loop gets the voltage for the module in the 0th index and sets it as the minimum value.